server/base/src/main/java/org/apache/accumulo/server/conf/util/ZooInfoViewer.java

```diff
@@ -231,13 +231,11 @@ private void printAcls(final ServerContext context, final Opts opts, final Print
     var conf = opts.getSiteConfiguration();
     try (var zk = new ZooSession(getClass().getSimpleName(), conf)) {
 
-      String instanceRoot = ZooUtil.getRoot(iid);
-
       final Stat stat = new Stat();
 
       recursiveAclRead(zk, ZROOT + ZINSTANCES, stat, aclMap);
 
-      recursiveAclRead(zk, instanceRoot, stat, aclMap);
+      recursiveAclRead(zk, "/", stat, aclMap);
```

Comment:
Not sure but suspect this behavior may differ.  Is this zookeeper chrooted?  If not is this reading all of zookeeper when it looks like it used to read a single instances data?

Reply:
You are correct. I fixed it in c6ab0db894bd47a2c3cb458e9753c29d9ea33e52